src/activities/content/ContentWebLinkEntity.js

```diff
@@ -0,0 +1,158 @@
+import { Entity } from '../../es6/Entity';
+import { Actions, Classes } from '../../hypermedia-constants';
+import { performSirenAction } from '../../es6/SirenAction';
+import ContentHelperFunctions from './ContentHelperFunctions.js';
+
+/**
+ * ContentWebLinkEntity class representation of a d2l content-weblink entity.
+ */
+export class ContentWebLinkEntity extends Entity {
+
+	/**
+	 * @returns {string} Description html of the content-weblink item
+	 */
+	descriptionRichText() {
+		const descriptionSubEntity = ContentHelperFunctions.getDescriptionSubEntity(this._entity);
+		if (!descriptionSubEntity) {
+			return null;
```

Comment:
same thing here. need to check if empty string is okay, or if returning `null|string` is acceptable

Reply:
speaking from experience in some of our other repos, I really dislike the fallback of returning an empty string. `null` actually means something, the empty strings just hide problems.

Reply:
especially lines like this:
```
return descriptionSubEntity.properties.html || '';
```
I think having a safety net is fine, but I'd rather have it moved up and make us return `null` instead as that be able to quickly tell us there's problem with the api "why didn't descriptionSubEntity have properties, or if it did, why didn't it have an html if we expect it to exist?"

Reply:
Agreed, hiding errors isn't ideal so we can just update the comments and handle the errors at the UI level.